packages/jest-core/src/cli/index.ts

```diff
@@ -32,7 +32,7 @@ import watch from '../watch';
 
 const {print: preRunMessagePrint} = preRunMessage;
 
-type OnCompleteCallback = (results: AggregatedResult) => void;
+type OnCompleteCallback = (results: AggregatedResult) => void | undefined;
```

Comment:
why o.O the `=> { results = r }` arrow function shouldn't require this, does it?

Reply:
we previously awaited this, which is not needed as the function is sync. But saying `void` is essentially same as `unknown`for the caller, so this ensure there's a type error if the function returns anything at all.

The change was due to it being `(results = r)`, meaning it returned it